### PR TITLE
iOS 10 support

### PR DESCRIPTION
**Pull request by lazyjean**

### CODE REVIEW

File: Source/PTEDashboard.m [hunk 1]
```diff
@@ -311,4 +311,9 @@ - (BOOL)prefersStatusBarHidden
     return NO;
 }
 
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    [self.tableView dataSource];
```
Comment (rivera-ernesto) [inline Source/PTEDashboard.m]: What is this for?

Comment (rivera-ernesto) [inline Source/PTEDashboard.m]: Now I see. #16 fixes it I think.

Can you give it a try?

**Comment by ppepingagne** [comment]
Any news ?

**Comment by rivera-ernesto** [comment]
Let's give it a try.

**Comment by misterpatthecat** [comment]
Seems good for me !

2016-11-18 10:30 GMT-05:00 Ernesto Rivera <notifications@github.com>:

**Pull request merged**